internal/commands/app/delete_inputs.go

```diff
@@ -0,0 +1,79 @@
+package app
+
+import (
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+var (
+	flagApps      = "apps"
+	flagAppsUsage = "the Realm app names or ids to manage"
+)
+
+type deleteInputs struct {
+	Apps    []string
+	Project string
+}
+
+func (inputs *deleteInputs) resolveApps(ui terminal.UI, client realm.Client) ([]realm.App, error) {
+	apps, err := client.FindApps(realm.AppFilter{GroupID: inputs.Project})
+	if err != nil {
+		return nil, err
+	}
+
+	if len(inputs.Apps) > 0 && len(apps) == 0 {
+		return nil, cli.ErrAppNotFound{}
+	}
+
+	if len(apps) == 0 {
```

Comment:
This was intentionally 0, since this would be hit if we found no apps and the user did not provide any inputs with the apps flag. It just serves as an early return. Here we still want to make prompt the user even is only 1 app is found so we don't automatically delete their app (which is why I modified this logic from ResolveApp a bit here)

Reply:
ah i see it now.... can we rearrange this bit then:

```go
if len(apps) == 0 {
  if len(i.Apps) == 0 {
    return nil, nil
  }
  return nil, cli.ErrAppNowFound{}
}
```